views/rss.blade.php

```diff
@@ -13,9 +13,7 @@
         <item>
             <title><![CDATA[{!! $entry['title'] !!}]]></title>
             <description><![CDATA[{!! $entry['description'] !!}]]></description>
-            <content:encoded><![CDATA[{!! $entry['content'] or $entry['description'] !!}]]></content:encoded>
-            <link>{{ $entry['permalink'] }}</link>
-            <guid isPermaLink="{{ !isset($entry['id']) }}">{{ $entry['id'] or $entry['permalink'] }}</guid>
+            <guid>{{ $entry['permalink'] }}</guid>
```

Comment:
We may want to add `isPermalin="true"` here.

```suggestion
            <guid isPermaLink="true">{{ $entry['permalink'] }}</guid>
```

Reply:
If the guid is a full URL, which this is, you don’t need the isPermaLink. See here: https://validator.w3.org/feed/docs/error/InvalidPermalink.html

Reply:
You're right, I misread the spec.